site/plugins/site/extensions/routes.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+use Kirby\Meta\SiteMeta;
+
+return [
+    /* Meta */
+    [
+        'pattern' => 'robots.txt',
+        'method' => 'ALL',
+        'action' => function () {
+            return SiteMeta::robots();
+        },
+    ],
+    [
+        'pattern' => 'sitemap.xml',
+        'action' => function () {
+            return SiteMeta::sitemap();
+        }
+    ],
+    [
+        'pattern' => 'open-search.xml',
+        'action' => function () {
+            return SiteMeta::search();
```

Comment:
Shouldn't this be added to the beginning of the base64 code? `data:image/png;base64,XXX`

Like that on related file: `site/plugins/site/src/Meta/snippets/search.php`
````xml
<Image width="64" height="64" type="image/png">data:image/png;base64,<?= (new Kirby\Toolkit\File(kirby()->root('index') . '/opensearch.png'))->base64() ?></Image>
````

Document:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs

Reply:
I know too little of XML to say whether it's needed there or not. I find different examples online. Migrated it as it is used currently on the website (without).

Reply:
Then let's leave it as it is 👍